src/browser/components/tabbrowser/content/tabbrowser-js.patch

```diff
@@ -103,3 +134,19 @@ index ef857bd81f2cd7c163ecc74ac1cf81a0b63ce838..807eb0c493f15643412b05d8dad81d36
          aTab.selected ||
          aTab.closing ||
          // Tabs that are sharing the screen, microphone or camera cannot be hidden.
+@@ -7870,7 +7925,14 @@ var TabContextMenu = {
+     );
+     contextUnpinSelectedTabs.hidden =
+       !this.contextTab.pinned || !multiselectionContext;
+-
++    let contextResetPinnedTab = document.getElementById("context_zen-reset-pinned-tab");
++    if(contextResetPinnedTab) {
```

Comment:
Same here, moving into a function inside the pinned tabs manager because large patches can end up being messy and could break in the future